### PR TITLE
HtmlFileAssignment

### PR DESCRIPTION
**Pull request by Pragati-3**

### CODE REVIEW

File: pragati/htmlDoc/about.css [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  color: purple;
+}
+h1 {
+  font-family: Helvetica, Geneva, Arial, SunSans-Regular, sans-serif;
+  color: black;
+}
+.new {
+  padding: 8px;
+  text-align: left;
+
```
Comment (KomalShinde813) [inline pragati/htmlDoc/about.css:12]: remove extra line

File: pragati/htmlDoc/aboutMe.html [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+    <link rel="stylesheet" href="about.css" />
+  </head>
+
+  <body>
+    <div class="about-section">
+      <h1>Pragati Zade</h1>
+      <p>pragati@gmail.com</p>
+    </div>
+    <div>
+      <h1>About Me</h1>
```
Comment (KomalShinde813) [inline pragati/htmlDoc/aboutMe.html:15]: check indentation need one tab space

File: pragati/htmlDoc/aboutMe.html [hunk 3]
```diff
@@ -0,0 +1,123 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
```
Comment (KomalShinde813) [inline pragati/htmlDoc/aboutMe.html:3]: check indentation in all over file give one tab space everywhere

File: pragati/htmlDoc/aboutMe.html [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+        My mother is a homemaker.<br />
+        My hobbies are making friends, listening to music, and watching
+        movies.<br />
+        I am looking forward to work in an MNC company. My aim is to become a
+        successful person in my life.
+      </p>
+    </div>
+    <div>
+      <h1>Work</h1>
+      <p>I have done my acdemic project in .net paltfrom</p>
```
Comment (KomalShinde813) [inline pragati/htmlDoc/aboutMe.html:28]: wrong spelling

File: pragati/htmlDoc/aboutMe.html [hunk 5]
```diff
@@ -0,0 +1,123 @@
...
+      <p>I am currting doing Internship on Mean Stack</p>
```
Comment (KomalShinde813) [inline pragati/htmlDoc/aboutMe.html:29]: wrong spelling

File: pragati/htmlDoc/aboutMe.html [hunk 6]
```diff
@@ -0,0 +1,123 @@
...
+      </ul>
+    </div>
+    <div>
+      <h3>Stuffs I would like to do someday</h3>
+      <ul type="squre">
+        <li>Confess my feelings to someone</li>
+        <li>Buy Own a Car</li>
+        <li>Spend a Night in Tree House</li>
+        <li>Want to settle down in Hyderabad</li>
+        <li>I want to tell Veryone dont touch my Mobile</li>
```
Comment (KomalShinde813) [inline pragati/htmlDoc/aboutMe.html:86]: wrong spelling

File: pragati/htmlDoc/aboutMe.html [hunk 7]
```diff
@@ -0,0 +1,123 @@
...
+      </ul>
+    </div>
+    <div class="new">
+      <h1>Contact</h1>
+      <form action="./aboutMe.html" target="_blank">
+        <label> Choose purpose of contact</label>
+        <select>
+          <option value="Official">Offical</option>
+          <option value="Persnal">Personal</option> </select
+        ><br />
```
Comment (KomalShinde813) [inline pragati/htmlDoc/aboutMe.html:96]: close this at upper line

File: pragati/htmlDoc/aboutMe.html [hunk 8]
```diff
@@ -0,0 +1,123 @@
...
+        <label>Name</label>
+        <input
+          type="text"
```
Comment (KomalShinde813) [inline pragati/htmlDoc/aboutMe.html:99]: write all attribute in one line

File: pragati/htmlDoc/aboutMe.html [hunk 9]
```diff
@@ -0,0 +1,123 @@
...
+          name="name"
+          tyle="font-size:16pt;"
```
Comment (KomalShinde813) [inline pragati/htmlDoc/aboutMe.html:101]: wrong spelling of style and need space after :

File: pragati/htmlDoc/aboutMe.html [hunk 10]
```diff
@@ -0,0 +1,123 @@
...
+          placeholder="Name"
+        /><br />
+        <label>Email</label>
+
```
Comment (KomalShinde813) [inline pragati/htmlDoc/aboutMe.html:105]: remove extra line

File: pragati/htmlDoc/aboutMe.html [hunk 11]
```diff
@@ -0,0 +1,123 @@
...
+        <input type="text" name="email" placeholder="Email" /><br />
+        <label>Message</label>
+
+        <textarea
+          type="text"
+          name="msg"
```
Comment (KomalShinde813) [inline pragati/htmlDoc/aboutMe.html:111]: write all attributes in one line

File: pragati/htmlDoc/about.css [hunk 12]
```diff
@@ -9,17 +9,14 @@ h1 {
 .new {
   padding: 8px;
```
Comment (anitapise) [inline pragati/htmlDoc/about.css:10]: need 1 tab space for all children follow the rule in every code files

File: pragati/htmlDoc/about.css [hunk 13]
```diff
@@ -0,0 +1,22 @@
+body {
+    font-family: Georgia, "Times New Roman", Times, serif;
+    color: purple;
+}
+h1 {
+  	font-family: Helvetica, Geneva, Arial, SunSans-Regular,sans-serif;
```
Comment (KomalShinde813) [inline pragati/htmlDoc/about.css:6]: remove extra space and space required after ,

File: pragati/htmlDoc/about.css [hunk 14]
```diff
@@ -0,0 +1,22 @@
...
+    color: black;
+}
+.new {
+     padding: 8px;
+     text-align: left;
+     color: black;
+}
+.about-section {
+       padding: 1px;
```
Comment (KomalShinde813) [inline pragati/htmlDoc/about.css:15]: check space for whole tab its not one tab

File: pragati/htmlDoc/aboutMe.html [hunk 15]
```diff
@@ -0,0 +1,113 @@
+<!DOCTYPE html>
+	<html lang="en-US">
```
Comment (KomalShinde813) [inline pragati/htmlDoc/aboutMe.html:2]: no need of space here its a parent tag for whole file

File: pragati/htmlDoc/aboutMe.html [hunk 16]
```diff
@@ -0,0 +1,113 @@
...
+					My mother is a homemaker.<br />
+					My hobbies are making friends, listening to music, and watching
+					movies.<br />
+					I am looking forward to work in an MNC company. My aim is to become a
+					successful person in my life.
+				  </p>
+			</div>
+			<div>
+				<h1>Work</h1>
+				  <p>I have done my academic project in .net platform</p>
```
Comment (KomalShinde813) [inline pragati/htmlDoc/aboutMe.html:27]: check space for above line  and this  tag inconsistent spacing

File: pragati/htmlDoc/aboutMe.html [hunk 17]
```diff
@@ -0,0 +1,113 @@
...
+				  <p>I am doing Internship on Mean Stack</p>
+			</div>
+			<div>
+			    <h1>EDUCATION</h1>
+			     <table border="1">
+					 <tr>
```
Comment (KomalShinde813) [inline pragati/htmlDoc/aboutMe.html:33]: how may tabs of space here?

File: pragati/htmlDoc/aboutMe.html [hunk 18]
```diff
@@ -0,0 +1,113 @@
...
+					    <th colspan="2">Institution</th>
+					    <th>Year of passing</th>
+					    <th>Certificate/Degree</th>
+					 </tr>
+					 <tbody>
+					    <tr>
```
Comment (KomalShinde813) [inline pragati/htmlDoc/aboutMe.html:39]: there is no one tab space here

File: pragati/htmlDoc/aboutMe.html [hunk 19]
```diff
@@ -0,0 +1,113 @@
...
+					</tbody>
+				 </table>
+			</div>
+			<div>	
+				<h2>MY TODO LIST</h2>
+				<h3>Stuffs I would like to do as soon as possible</h3>
+				 <ul type="circle">
+					     <li>I want get best job in IT company</li>
+					     <li>I want visit USA.</li>
+						 <li>I want become Angular Developer </li>
```
Comment (KomalShinde813) [inline pragati/htmlDoc/aboutMe.html:72]: inconsistent spacing

File: pragati/htmlDoc/aboutMe.html [hunk 20]
```diff
@@ -0,0 +1,113 @@
...
+						 <td>HSC</td>
+					   </tr>
+					</tbody>
+				 </table>
+			</div>
+			<div>	
+				<h2>MY TODO LIST</h2>
+				<h3>Stuffs I would like to do as soon as possible</h3>
+				 <ul type="circle">
+					     <li>I want get best job in IT company</li>
```
Comment (KomalShinde813) [inline pragati/htmlDoc/aboutMe.html:70]: how many tab of space here?

File: pragati/htmlDoc/aboutMe.html [hunk 21]
```diff
@@ -0,0 +1,113 @@
...
+					     <li>I want visit USA.</li>
+						 <li>I want become Angular Developer </li>
+						 <li>I want to best life for my parents.</li>
+						 <li>I want meet Sidharth Shukla</li>
+				 </ul>
+			</div>
+			<div>
+				<h3>Stuffs I would like to do someday</h3>
+				<ul type="squre">
+					     <li>Confess my feelings to someone</li>
```
Comment (KomalShinde813) [inline pragati/htmlDoc/aboutMe.html:80]: how many tab of space here?

File: pragati/htmlDoc/aboutMe.html [hunk 22]
```diff
@@ -0,0 +1,113 @@
...
+				<form action="./aboutMe.html" target="_blank">
+					<h1>Contact</h1>
+					<label> Choose purpose of contact </label>
+					<select>
+						<option value="Persnal">Personal</option>
+						<option value="Official">Offical</option>
+					</select>	
+					<br>
+					<label>Name</label>
+					<input type="text" name="name" style="font-size: 15px;"  placeholder="Name"	/>
```
Comment (KomalShinde813) [inline pragati/htmlDoc/aboutMe.html:97]: remove extra space after style and before closing tag

File: pragati/htmlDoc/aboutMe.html [hunk 23]
```diff
@@ -0,0 +1,113 @@
...
+					<br />
+				    <label>Email</label>
+				    <input type="text" name="email" style="font-size: 15px;" placeholder="Email" />
+					<br />
```
Comment (KomalShinde813) [inline pragati/htmlDoc/aboutMe.html:101]: space is not required

File: pragati/htmlDoc/aboutMe.html [hunk 24]
```diff
@@ -0,0 +1,113 @@
...
+				    <label>Message</label>
+					<textarea type="text" name="msg" placeholder="Write Something Here..."></textarea>
```
Comment (KomalShinde813) [inline pragati/htmlDoc/aboutMe.html:103]: space is not required check spacing everywhere

File: pragati/htmlDoc/aboutMe.html [hunk 25]
```diff
@@ -0,0 +1,109 @@
+<!DOCTYPE html>
+<html lang="en-US">
+  <head>
+    <title>About</title>
+    <link rel="stylesheet" href="about.css" />
+  </head>
+  <body>
+    <div class="about-section">
```
Comment (KomalShinde813) [inline pragati/htmlDoc/aboutMe.html:8]: 4 spaces required

**Pull request merged**